events/wc_void_events.txt

```diff
@@ -1578,6 +1575,10 @@ character_event = {
 			factor = 1.2
 			is_mage_class_trigger = yes
 		}
+		modifier = {
+			factor = 5
+			controls_old_god_living_prison_trigger = yes
+		}
```

Comment:
So, since it uses `has_wonder` trigger, it increases the chance for a top liege.
Thus, for example, Lord Admiral of Kul Tiras is in danger, the High Sea Priest (who holds the Shrine of Storms) isn't.

Reply:
so should it work for owner or holder of the province?

Reply:
> so should it work for owner or holder of the province?

It should work for the High Sea Priest.

Reply:
fixed